pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotTaskManager.java

```diff
@@ -163,6 +164,9 @@ public Map<String, String> createTask(String taskType, String tableName, @Nullab
         taskConfigs.getOrDefault(MINION_INSTANCE_TAG_CONFIG, CommonConstants.Helix.UNTAGGED_MINION_INSTANCE);
     _helixTaskResourceManager.ensureTaskQueueExists(taskType);
     addTaskTypeMetricsUpdaterIfNeeded(taskType);
+    if (!isTaskSchedulable(taskType, List.of(tableName))) {
```

Comment:
If the task queue does not exist before, the method `ensureTaskQueueExists` creates the queue and starts the queue. In this case, we do not want to prevent task creation. For this, `isTaskSchedulable` is kept after the method `ensureTaskQueueExists`.